db/migrate/20150826191632_create_target_remote_execution_proxies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTargetRemoteExecutionProxies < ActiveRecord::Migration
+  def change
+    create_table :target_remote_execution_proxies do |t|
+      t.integer :remote_execution_proxy_id
```

Comment:
Indexes on foreign keys can help us later (on `:remote_execution_proxy_id` and `[:target_id, :target_type]`, especially if somebody used the per-host target association